### ISSUE TITLE
Conversion is adding `label-color` when none should be present and not honoring `label-scale`

### ISSUE THREAD

**Issue by kthyer** [placemark/togeojson#75]
I am using  Google My Maps and I have noticed in the newer versions of this conversion tool, it is now adding a `label-color` that matches the `icon-color` when it should not be present. It did not used to do this in previous versions.  It also is now not reading in the `label-scale` property when it used to as well.

I forked your online tool so that it used the latest version of this library and that is where you can reproduce what I'm seeing by uploading the KML files I have provided in the ZIP file below.

The fork can be found here: https://observablehq.com/d/96ff9e6b1b6edfb1

This is a snippet of the attached KML file below
```Style id="icon-1899-0288D1-nodesc-normal">
      <IconStyle>
        <color>ffd18802</color>
        <scale>1</scale>
        <Icon>
          <href>https://www.gstatic.com/mapspro/images/stock/503-wht-blank_maps.png</href>
        </Icon>
        <hotSpot x="32" xunits="pixels" y="64" yunits="insetPixels"/>
      </IconStyle>
      <LabelStyle>
        <scale>0</scale>
      </LabelStyle>
      <BalloonStyle>
        <text><![CDATA[<h3>$[name]</h3>]]></text>
      </BalloonStyle>
    </Style>
```

When I import using the latest conversion tool I get

![image](https://user-images.githubusercontent.com/850356/164516489-8c6cd9c0-8f19-4770-a87c-a5d21d803085.png)

As you can see, it is adding the `label-color` to match the `icon-color` while also not adding the `label-scale` at all.

When uploading to a previous version it converts to
![image](https://user-images.githubusercontent.com/850356/164516824-3a2c466e-9b20-44dd-abda-ac71df6bb2fb.png)

It properly gives no `label-color` and sets the `label-scale` to 0

[Google My Maps Placemark.kml.zip](https://github.com/placemark/togeojson/files/8533239/Google.My.Maps.Placemark.kml.zip)

Thanks and let me know if I can clarify anything else

**Comment by tmcw** [comment]
Good catch! There was a bug in style parsing that was looking for any color node, not just those within that part of the style. Fix incoming.

**Comment by tmcw** [comment]
Fixed & released in 5.0.0.
